src/test/java/com/google/cloud/spanner/pgadapter/PgAdapterTestEnv.java

```diff
@@ -151,10 +151,8 @@ public String getCredentials() {
   }
 
   public int getPort() {
-    if (port == 0) {
-      Random rand = new Random(System.currentTimeMillis());
-      int defaultPort = rand.nextInt(10000) + 10000;
```

Comment:
Do not use random number as port number, as there is no guarantee that this port number is not used by some other service on the host machine, even if it is a port number higher than 10,000. Instead we should use port number 0, as that instructs the OS to assign a randomly chosen __available__ port number.